ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Test the connectivity for {{ apivip }} 
+  command: ping -c3 -i 3 {{ apivip }} 
+  register: apivip_results
+  tags: vip_status
+
+- name: Test the connectivity for {{ ingressvip }}
```

Comment:
Can we change this to "Test for existing Ingress VIP connectivity" -- Lets not use the variable {{ ingressvip }} because that will show an IP and the user may not be aware directly this is what we are testing.